src/SectionBuilder.php

```diff
@@ -34,12 +34,13 @@ public function build($identifier, $value)
 
     private $data = [
         '00' => [18, 18, 'SERIAL SHIPPING CONTAINER CODE (00) - must contains exact 18 digits'],
-        '01' => [14, 14, 'GLOBAL TRADE ITEM NUMBER (01) - must contains excat 14 digits'],
+        '01' => [8, 14, 'GLOBAL TRADE ITEM NUMBER (01) - must contains excat 14 digits'],

```

Comment:
Not sure if I understand this change. According to 
https://www.gs1.org/standards/barcodes/application-identifiers/01?lang=en
01 is always 14 chars long

Reply:
Fuck me, you're totally correct. I didn't notice that for GTIN8, 12, 13, you have to prepend them with 0's to get to 14 characters 🤦 
So the only change left is the AI 17 then
Thanks for doing a great job!